File: problemas/ad-hoc/2414/2414.rs
use std::io;
use std::cmp::max;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let numeros: Vec<i32> = input.split_whitespace().map(|s| s.parse().expect("")).collect();

    let mut maior = numeros[0];
    for i in 1..numeros.len() {
        maior = max(maior, numeros[i]);
    }

    println!("{}", maior);
}

File: problemas/ad-hoc/2414/2414.dart
import 'dart:io';
import 'dart:math';

void main() {
  List<int> numeros =
      stdin.readLineSync()!.split(' ').map((x) => int.parse(x)).toList();

  int maior = numeros[0];
  for (int i = 1; i < numeros.length; i++) {
    maior = max(maior, numeros[i]);
  }

  print(maior);
}
